lib/reqTemplate.js

```diff
@@ -416,6 +414,10 @@ function Template(origSpec, globalsInit) {
         }
         var ret = res;
         res = null;
+        // ensure a reasonable fallback for ret.method
+        if (ret.hasOwnProperty('method')) {
```

Comment:
`ret` might be undefined, for example when we expand some `options` object while the `options` for the module were not configured.